includes/Core/Authentication/Authentication.php

```diff
@@ -1353,4 +1353,27 @@ private function filter_features_via_proxy( $feature_enabled, $feature_name ) {
 		return $feature_enabled;
 	}
 
+	/**
+	 * Invalid nonce error handler.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $action    Action name.
+	 */
+	public function invalid_nonce_error( $action ) {
+		if ( strpos( $action, 'googlesitekit_proxy_' ) === 0 ) {
```

Comment:
Let's return early if the action doesn't start with `googlesitekit_proxy_`:

```php
if ( strpos( $action, 'googlesitekit_proxy_' ) === false ) {
    wp_nonce_ays( $action );
    return;
}

// Copied from wp_nonce_ays() with tweak to the url.
$html = __( 'The link you followed has expired.', 'default' );
...
```

Reply:
https://github.com/google/site-kit-wp/pull/3976/commits/ef254b84ba25533d0a89d04a5b637a3450a5ebe5